src/styles/blocks/card.scss

```
@@ -0,0 +1,104 @@
+.card {
+  box-sizing: border-box;
+  background-color: $white;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  @include hover (transform, scale(1.2))
+
+    &:hover &__title {
+    transition: $transitionDefault;
+    color: #34568b;
+  }
+
+  &__image {
+    box-sizing: content-box;
+    margin: auto;
+    width: 160px;
+    height: 134px;
+    padding: 16px 0 40px;
+  }
+
+  &__title {
+    color: $mainAccentFontColor;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    padding: 4px 0;
+    color: $secondaryFontColor;
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+  }
+  
+  &__reviews {
+    padding: 3px 0;
```

Maybe 4? Even if we have broken indent try to use number divisible by 4
```suggestion
    padding: 3px 0;
```

---

ok, but 4 px does not pass the tests